rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -659,8 +659,8 @@ mod tests {
 
     #[rstest]
     #[case(4, DistanceType::L2, 0.9)]
-    #[case(4, DistanceType::Cosine, 0.6)]
-    #[case(4, DistanceType::Dot, 0.2)]
+    #[case(4, DistanceType::Cosine, 0.9)]
+    #[case(4, DistanceType::Dot, 0.9)]
```

Comment:
Why did these results change?  Shouldn't transposing not affect the calculation?

Reply:
not related to transposing
I think IVF_PQ can always reach this recall, these tests were added when working on HNSW, which would result in low recall with the other distance types, looks like I copied HNSW's tests and forgot to modify the thresholds for IVF_PQ